subsys/bluetooth/services/fast_pair/fp_storage/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-add_library(fp_storage STATIC)
+zephyr_library_named(fp_storage)
```

Comment:
After this is applied, the `target_link_libraries` at the bottom of this file is no longer needed.
Please apply the same change to the `fp_crypto` helper library too

Reply:
done